tests/importer/importer.cpp

```diff
@@ -879,6 +879,95 @@ TEST(Importer, clearModelImportsBeforeResolving)
     EXPECT_EQ(size_t(0), importer->issueCount());
 }
 
+TEST(Importer, isResolvedUnitsNoModel)
```

Comment:
a bunch of these tests are testing if imports are required, rather than whether imports are resolved. Not really a big deal, but perhaps worth flagging the inconsistency of the test names for future revision?

Reply:
I will make this change in the next PR so this one can be merged without further review.